### PR TITLE
SD3: update default training timestep / loss weighting distribution to logit_normal

### PR DESCRIPTION
**Pull request by bghira**
# What does this PR do?

Though it's not as obvious when training on small batch sizes, it's really obvious at large batch sizes that the timestep distribution is not correct.

When reflecting upon the SD3 paper, I saw that they use `rf/lognorm(0,1)` as their ideal weighting formulation.

It doesn't really do much to save the model or make it any more easily trainable, unfortunately. But, it keeps it from collapsing. Might seem like that contradicts what I just said, but people who have tried training SD 2.0 will know what I mean.

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [ ] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)? Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/diffusers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by kashif** [comment]
yeah. looks good to me!

**Comment by Slickytail** [comment]
Yep, I agree. This should absolutely be the default setting, since it's the one that SD3 was trained with.

**Comment by sayakpaul** [comment]
Failing tests are unrelated.
